### PR TITLE
#39 Use cactoos FormattedText instead of static String format.

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Let's use OOP primitives from cactoos for string formatting.
As per #39

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is now in scope, role is `REV`

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
Job [#40](https://github.com/yegor256/cactoos-http/issues/40) is already in scope

**Comment by 0crat** [comment]
This pull request [#40](https://github.com/yegor256/cactoos-http/issues/40) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C63314D6Z/1222130511); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/test/java/org/cactoos/http/HtAutoRedirectTest.java [hunk 1]
```diff
@@ -53,7 +56,11 @@ public void redirectsRequestAutomatically() throws Exception {
                             new JoinedText(
                                 "\r\n",
                                 "HTTP/1.1 301",
-                                String.format("Location: %s", home)
+                                new UncheckedText(
```
Comment (krzyk) [inline src/test/java/org/cactoos/http/HtAutoRedirectTest.java]: @alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `Exception`

File: src/test/java/org/cactoos/http/HtResponseTest.java [hunk 2]
```diff
@@ -53,7 +55,11 @@ public void worksFine() throws IOException {
                         new JoinedText(
                             "\r\n",
                             "GET / HTTP/1.1",
-                            String.format("Host:%s", home.getHost())
+                                new UncheckedText(
```
Comment (krzyk) [inline src/test/java/org/cactoos/http/HtResponseTest.java]: @alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

File: src/test/java/org/cactoos/http/HtSecureWireTest.java [hunk 3]
```diff
@@ -132,7 +134,12 @@ public InputStream stream() throws IOException {
                 new JoinedText(
                     delimiter,
                     "GET / HTTP/1.1",
-                    String.format("Host: %s", this.host),
+                    new UncheckedText(
```
Comment (krzyk) [inline src/test/java/org/cactoos/http/HtSecureWireTest.java]: @alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

File: src/test/java/org/cactoos/http/HtWireTest.java [hunk 4]
```diff
@@ -78,7 +80,12 @@ public void worksWithProvidedHostNameAndPort() throws IOException {
                         new JoinedText(
                             "\r\n",
                             "GET / HTTP/1.1",
-                            String.format("Host:%s", home.getHost())
+                            new UncheckedText(
```
Comment (krzyk) [inline src/test/java/org/cactoos/http/HtWireTest.java]: @alex-semenyuk Why `UncheckedText` here? this test already declares it throws an `IOException`

File: src/test/java/org/cactoos/http/HtAutoRedirectTest.java (hunk 1 above)
Comment (alex-semenyuk) [reply src/test/java/org/cactoos/http/HtAutoRedirectTest.java]: Fixed

File: src/test/java/org/cactoos/http/HtResponseTest.java (hunk 2 above)
Comment (alex-semenyuk) [reply src/test/java/org/cactoos/http/HtResponseTest.java]: Fixed

File: src/test/java/org/cactoos/http/HtSecureWireTest.java (hunk 3 above)
Comment (alex-semenyuk) [reply src/test/java/org/cactoos/http/HtSecureWireTest.java]: Fixed

File: src/test/java/org/cactoos/http/HtWireTest.java (hunk 4 above)
Comment (alex-semenyuk) [reply src/test/java/org/cactoos/http/HtWireTest.java]: Fixed

**Comment by krzyk** [comment]
@alex-semenyuk Looks good, for the QA the codecov/patch diff target is a mistake, as you can see in the project code coverage it doesn't change the code coverage and doesn't introduce new paths

**Comment by krzyk** [comment]
@rultor merge

**Comment by krzyk** [comment]
@llorllale Hmm, rultor should ask you here, strange. I approve this PR so you can take a look.

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#40](https://github.com/yegor256/cactoos-http/issues/40) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by elenavolokhova** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**